static/Makefile

```diff
@@ -36,83 +16,23 @@ help: ## show make targets
 clean: ## remove build artifacts
 	[ ! -d build ] || $(CHOWN) -R $(shell id -u):$(shell id -g) build
 	$(RM) -r build
-	-docker builder prune -f --filter until=24h
-
-.PHONY: static
-static: static-linux cross-mac cross-win cross-arm ## create all static packages
 
-.PHONY: static-linux
-static-linux: static-cli static-engine static-buildx-plugin ## create tgz
-	mkdir -p build/linux/docker
-	cp $(CLI_DIR)/build/docker build/linux/docker/
-	for f in dockerd containerd ctr containerd-shim containerd-shim-runc-v2 docker-init docker-proxy runc; do \
-		cp -L $(ENGINE_DIR)/bundles/binary-daemon/$$f build/linux/docker/$$f; \
-	done
-	tar -C build/linux -c -z -f build/linux/docker-$(GEN_STATIC_VER).tgz docker
-
-	# extra binaries for running rootless
-	mkdir -p build/linux/docker-rootless-extras
-	for f in rootlesskit rootlesskit-docker-proxy dockerd-rootless.sh dockerd-rootless-setuptool.sh vpnkit; do \
-		if [ -f $(ENGINE_DIR)/bundles/binary-daemon/$$f ]; then \
-			cp -L $(ENGINE_DIR)/bundles/binary-daemon/$$f build/linux/docker-rootless-extras/$$f; \
-		fi \
-	done
-	tar -C build/linux -c -z -f build/linux/docker-rootless-extras-$(GEN_STATIC_VER).tgz docker-rootless-extras
-
-	# buildx
-	tar -C $(BUILDX_DIR)/bin -c -z -f build/linux/docker-buildx-plugin-$(DOCKER_BUILDX_REF:v%=%).tgz docker-buildx
+.PHONY: static-%
+static-%:
+	PRODUCT="$(PRODUCT)" \
+	PLATFORM="$(PLATFORM)" \
+	DEFAULT_PRODUCT_LICENSE="$(DEFAULT_PRODUCT_LICENSE)" \
```

Comment:
Slightly related; so I know there's cases where "not set" vs "empty" makes a difference; specifically when using `--build-arg`, where "not set" means "use defaults from the Dockerfile" and "empty" means "use an empty string". Is that a concern here?

Reply:
Yes it would be better if we could pass makefile vars as envs. Do you know a way to do this?